Revert this commit:

commit 73cc289e7932e288cf9aed82b1d6e82ea5615c16
Author: Mark Andrews <marka@isc.org>
Date:   2017-08-05 12:31:45 +1000

    remove unused variable 'fctx' from rctx_next

--- lib/dns/resolver.c.orig	2017-08-05 02:36:31 UTC
+++ lib/dns/resolver.c
@@ -8939,6 +8939,7 @@ rctx_resend(respctx_t *rctx, dns_adbaddr
  */
 static void
 rctx_next(respctx_t *rctx) {
+	fetchctx_t *fctx = rctx->fctx;
 	isc_result_t result;
 
 	FCTXTRACE("nextitem");
